vogue/commands/load/base.py

```diff
@@ -11,11 +15,23 @@
 
 @click.group()
 @click.version_option(version=__version__)
+@click.option('-d', '--database-name', default = 'trending', show_default = True, 
```

Comment:
Let's make this a click choice:
```suggestion
@click.option('-d', '--database-name', type=click.Choice(['trending', 'trending_stage']), default = 'trending', show_default = True, 
```